### ISSUE TITLE
Enhancement request: Reproducible results

### ISSUE THREAD

**Issue by ivan-marroquin** [stekhoven/missForest#21]
Hi

Thanks for making available such great tool. I noticed that for the upcoming developments, there is the plan to support reproducible imputation results. I believe that is a very important step forward. 

Ivan

**Comment by jwe4ec** [comment]
I agree. A colleague and I are getting different results on different operating systems despite setting the same seed via `set.seed()` prior to running `missForest()` and despite running the same R version (`4.4.0`) and ensuring the same package versions are loaded (`missForest_1.5`, including the same versions of its dependencies). My colleague is getting the error below on `macOS Sonoma 14.7.5`, whereas I'm getting no error on `Windows 11 x64 (build 22631)`.

```r
set.seed(12345)
dat_imputed <- missForest(dat)
```
Here is the error my colleague is getting on macOS:
<img width="1637" height="156" alt="Image" src="https://github.com/user-attachments/assets/47c1bc19-7428-4100-982d-0f6145cc775f" />

Here are our two `sessionInfo()` outputs:
[sessionInfo_macOS.txt](https://github.com/user-attachments/files/21781570/sessionInfo_macOS.txt)
[sessionInfo_Windows.txt](https://github.com/user-attachments/files/21781640/sessionInfo_Windows.txt)

We would very much appreciate any advice you might have for obtaining reproducible results across operating systems, given how important computational reproducibility is in science!
